### DISCUSSION TITLE
Return the description of panels in the api

### DISCUSSION THREAD

**Discussion by PsclDev** [grafana/grafana#53492] [Dashboards, Panels & Visualizations]
It would be nice to get also the panel description from the api and not just the title

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
